velox/docs/monthly-updates/november-2023.rst

```diff
@@ -0,0 +1,102 @@
+********************
+November 2023 Update
+********************
+
+Core Library
+============
+
+* Add support to export constant vector as `Arrow REE
+  array <https://arrow.apache.org/docs/format/Columnar.html#run-end-encoded-layout>`_. :pr:`7327`, :pr:`7398`
+* Add spilling support for distinct aggregation. :pr:`7305`
```

Comment:
Let's combine these 2: 

Add spilling support for aggregations over distinct or sorted inputs.